geometric/tests/test_qcengine.py

```diff
@@ -41,3 +42,81 @@ def test_rdkit_simple(localizer):
     # Currently in angstrom
     ref = np.array([0., 0., -0.0644928042, 0., -0.7830365196, 0.5416895554, 0., 0.7830365196, 0.5416895554])
     assert np.allclose(ref, ret.xyzs[-1].ravel(), atol=1.e-5)
+
```

Comment:
Have you confirmed that these tests can run locally?  How long do the tests take?

Reply:
Yes, it's not much proof but see below. They take about a minute in parallel (and ordinarily one wouldn't have psi4, nwchem, and cfour all accessible). At the moment they need software versions that are unreleased.
```
> pytest -v geometric/tests/test_qcengine.py -n 8
================================================================= test session starts ==================================================================
platform linux -- Python 3.12.2, pytest-8.1.1, pluggy-1.4.0 -- /psi/toolchainconda/envs/qcmb312_pyd1/bin/python3.12
cachedir: .pytest_cache
rootdir: /psi/gits/geomeTRIC
configfile: pytest.ini
plugins: xdist-3.5.0, typeguard-4.2.1
8 workers [7 items]     
scheduling tests via LoadScheduling

geometric/tests/test_qcengine.py::test_lif_bsse[cfour-nocp] 
geometric/tests/test_qcengine.py::test_lif_bsse[nwchem-cp] 
geometric/tests/test_qcengine.py::test_rdkit_simple 
geometric/tests/test_qcengine.py::test_lif_bsse[psi4-cp] 
geometric/tests/test_qcengine.py::test_lif_bsse[nwchem-nocp] 
geometric/tests/test_qcengine.py::test_lif_bsse[psi4-nocp] 
geometric/tests/test_qcengine.py::test_lif_bsse[cfour-cp] 
[gw0] [ 14%] PASSED geometric/tests/test_qcengine.py::test_rdkit_simple 
[gw3] [ 28%] PASSED geometric/tests/test_qcengine.py::test_lif_bsse[cfour-nocp] 
[gw4] [ 42%] PASSED geometric/tests/test_qcengine.py::test_lif_bsse[cfour-cp] 
[gw1] [ 57%] PASSED geometric/tests/test_qcengine.py::test_lif_bsse[psi4-nocp] 
[gw2] [ 71%] PASSED geometric/tests/test_qcengine.py::test_lif_bsse[psi4-cp] 
[gw5] [ 85%] PASSED geometric/tests/test_qcengine.py::test_lif_bsse[nwchem-nocp] 
[gw6] [100%] PASSED geometric/tests/test_qcengine.py::test_lif_bsse[nwchem-cp] 

=================================================================== warnings summary ===================================================================
  /psi/gits/geomeTRIC/geometric/tests/addons.py:51: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    from pkg_resources import parse_version

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================================================= 7 passed, 8 warnings in 68.03s (0:01:08) =======================================================
```